src/sass/partials/global/_extends.scss

```diff
@@ -0,0 +1,26 @@
+.base--a {
```

Comment:
We are not using extends. These have styling elsewhere (in base) that you should use